clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -149,6 +185,29 @@ Queue<StagedCommit> stagedCommits() {
         return this.stagedCommits;
     }
 
+    /**
+     * Get all the sendable requests, and create a list of UnsentRequest.
+     */
+    private List<NetworkClientDelegate.UnsentRequest> sendOffsetFetchRequests(final long currentTimeMs) {
+        List<UnsentOffsetFetchRequestState> requests = unsentOffsetFetchRequests.sendableRequests(currentTimeMs);
+        List<NetworkClientDelegate.UnsentRequest> pollResults = new ArrayList<>();
```

Comment:
I could also do it in request.stream().map().reduce().  Let me know if one way is better than the other.